modules/ROOT/pages/_partials/gs-deploying-apicast-proc.adoc

```diff
@@ -0,0 +1,37 @@
+[id='gs-deploying-apicast-proc']
+
+ifdef::env-github[]
+:imagesdir: ../images/
+endif::[]
+
+// https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.5/html/installing_3scale/installing-apicast
+
+= Deploying an APIcast API gateway
+
+APIcast is an NGINX based API gateway used to integrate your internal and external API services with the 3scale Platform.
+
+There are two APIcast deployment options:
+
+*Embedded APIcast*: {PRODUCT_SHORT} hosts APIcast in the cluster. In this case, APIcast is already deployed for you, making it easy to get started with development, but is not suitable for a production environment.
+
+*Self-managed APIcast*: You can deploy APIcast wherever you want. Red Hat recommends using a self-managed deployment for product.
```

Comment:
@evanshortiss  is this true? RH recommend  provisioning an apicast for production?
or just a hangover from their hosted docs?

Reply:
Hangover from hosted/SaaS docs IIRC. Andrew M can clarify though.